eclipse-collections/src/main/java/org/eclipse/collections/impl/list/immutable/ImmutableArrayList.java

```diff
@@ -501,10 +501,9 @@ public String toString()
         buf.append('[');
 
         int localSize = this.items.length;
-        T[] localItems = this.items;
         for (int i = 0; i < localSize; i++)
         {
-            T item = localItems[i];
+            T item = this.items[i];
```

Comment:
If @donraab is ok with inlining these. Otherwise, I'd leave the inspection as a warning in the editor, remove it from profile 4 completely, and neither fix nor suppress instances like this.

Reply:
TBH I did not quite understand why we had a local copy created. Since it is immutable, I would expect the array not change, or may be I am missing something.
@donraab thoughts?

Reply:
Write some JMH tests.  If there is no or very little performance difference, please go ahead with the change.